### ISSUE TITLE
Nimodipine no longer in workflow query C.2 results

### ISSUE THREAD

**Issue by rtroper** [NCATSTranslator/minihackathons#278]
In April 2021, query C.2 results (from ARAX) included nimodipine, a particularly interesting result: https://arax.ncats.io/beta/?r=8461. However, this result no longer shows up. It was thought this may be due to updates/issues in the ARAX ranking algorithm, prompting the following issue in the RTX repo: https://github.com/RTXteam/RTX/issues/1627. Subsequently, the follow issue (in ranking) was found: https://github.com/RTXteam/RTX/issues/1630.

Both issue have been fixed and nimodipine still does not appear in the top 100 ARAX results. As Amy Glen noted (https://github.com/RTXteam/RTX/issues/1627#issuecomment-906899598), nimodipine is well-connected: https://arax.ncats.io/?r=23288. Furthermore, she notes that less well-connected results show up in the top 100. 

Given Amy's findings, this warrants further investigation. Adding this issue here to keep attention on it.

**Comment by rtroper** [comment]
An update on this - the best I've managed to do is get nimodipine to show up as result number 2414. And this result I found shows up as "(s)-nimodipine" and doesn't have many genes connected to it (1 or 2). It doesn't appear to be the same result that Amy tracked down when she specified nimodipine for node 3. There seem to be some other interesting results. We might need to give up on the nimodipine result of yore....
